.github/workflows/dependabot.yml

```diff
@@ -0,0 +1,30 @@
+# Automatically approve dependabot PRs.
+
+name: Dependabot
+
+on:
+  - pull_request_target
+
+jobs:
+  approve:
+    name: Approve Dependabot PRs
+    runs-on: ubuntu-latest
+    if: github.actor == 'dependabot[bot]'
+
+    # IMPORTANT: setting YARN_ENABLE_SCRIPTS=false is critical to ensure that untrusted
+    # PRs can't add an npm package and then use that to execute untrusted code in
+    # a trusted context. See links at the top of this workflow for further details.
```

Comment:
Outdated commented? There are no links at the top of this workflow

Reply:
Yea - the links part is outdated but it seemed nice to keep this env var as a precaution? Though this PR doesn't run yarn anymore now that I think about it